themes/_default.yaml

```diff
@@ -1,6 +1,8 @@
 generic:
   border:
     radius: &border-radius 8px
+    width: &border-width 2px
```

Comment:
it's easy way to use the same border size everywhere, instead of hardcode the size.
also easy to customize and refactor.

Reply:
I'm not sure that good idea to hardcode border width, because sometimes people use 2px sometimes 3px

Reply:
at this moment, we are using the same border size for the following components(hardcoded value for all of these):
- autocomplete
- button
- card
- checkbox
- datepicker
- dropdown
- input
- radioButton
- toggleButton

I don't know why the variable still does not created.

Reply:
@EduardTrutsyk but the question is, are they related? When one changes, should all change?

Not saying that you're wrong, but if you don't know the answers to the questions above (e.g. I don't 😉 😄 ) and you plan to change them, please ask the designers if all those borders are related.

Otherwise, if they're not, you'll have a problem if all use the same variable or custom variables dependent on a main one.

Hardcoding makes sense when the value of the _possible_ variable would not be used/shared amongst components (therefore, as @iwwwi said, not overwritten).

Reply:
@mAiNiNfEcTiOn true, but:
1. sometimes raw design has mistakes, and when I notice, that the border size is 3px but in common variable we have 2px, then I will double check with designer, because I need to override it. but maybe it was designers mistake, and they will say: "no, it should be 2px". In this case, we have communication with designers. but when we don't have such common variable, I will just use 3px, like in design.
2. I'm not sure that all guys check the design attentively, or sometimes we don't have design. in this case, developer will use common variables instead of random values.

just my opinion, but if you do not think so, I will delete it.

Reply:
btw, on design markups for these component we have the same border size, for some of these components we don't have design.